client/src/main/java/io/pravega/client/tables/KeyValueTableConfiguration.java

```diff
@@ -36,4 +36,25 @@
      * @return The number of Partitions for a Key-Value Table.
      */
     private final int partitionCount;
+
+    /**
+     * The number of bytes for the Primary Key. This value cannot be changed after the Key-Value Table has been created.
+     *
+     * @param primaryKeyLength The number of bytes for the Primary Key.
+     * @return The number of bytes for the Primary Key.
+     */
+    private final int primaryKeyLength;
+
+    /**
+     * The number of bytes for the Secondary Key. This value cannot be changed after the Key-Value Table has been created.
+     *
+     * @param secondaryKeyLength The number of bytes for the Secondary Key.
+     * @return The number of bytes for the Primary Key.
+     */
+    private final int secondaryKeyLength;
```

Comment:
Because these are set by a builder they are implicitly optional and as no default value is provided they will all default to `-1`. I don't think this is desireable.

Reply:
We want them to be explicitly set by the user. When the Controller portion is done, it will have validation for these values, both on the client side and Controller side.

Reply:
Fixed